hugo/layouts/community/single.html

```diff
@@ -5,92 +5,30 @@
-                    </div> -->
-                    <div class="col-md mb-3 mb-md-0">
-                        <!-- <input type="text" class="form-control" name="email" placeholder="margaret@nasa.com"> -->
-                        &nbsp;
-                    </div>
-                    <div class="col-md">
-                        <button type="submit" class="btn btn-highlight w-100">Sign Up</button>
-                    </div>
-                    <div class="col-md mb-3 mb-md-0">
-                            &nbsp;
-                        </div>
-                </div>
-            </form>
-            <p class="text-center text-white mt-4">We’re happy to answer any questions you might have about  community, help you find the right resources, and meet other Machine Learning on Code enthusiasts from around the world!</p>
-        </div>
-    </div>
-</section>
-
-<section id="talk" class="pt-5 pb-5">
+<section id="channels" class="py-5">
     <div class="container text-dark">
         <div class="row">
             <div class="col-md-6">
-                <h2 class="title-line line-yellow">Talk to us</h2>
-                <p class="text-large">Join us on Slack to learn, contribute and collaborate with other developers who are passionate about the <strong>Code as Data</strong> and <strong>Machine learning on Code</strong> movements.</p>
+                <h2 class="title-line line-yellow">Our Community Channels</h2>
+                <p>Join us on the following channels to learn, contribute and collaborate with other executives, IT managers or Developers who are passionate about Code Analysis,  Engineering Effectiveness, Software Development Analytics and Machine Learning on Code.</p>
             </div>
             <div class="col-md-6">
-                <div class="card text-dark light">
+                <div class="card light">
                     <div class="card-body p-5">
-                        <a href="https://join.slack.com/t/sourced-community/shared_invite/enQtMjc4Njk5MzEyNzM2LTFjNzY4NjEwZGEwMzRiNTM4MzRlMzQ4MmIzZjkwZmZlM2NjODUxZmJjNDI1OTcxNDAyMmZlNmFjODZlNTg0YWM" class="ic icon-slack text-light"></a>
-                        <h4 class="title-line line-yellow m-0">Join Our Community</h4>
-                        <p>Join hundreds of developers passionate about Code as Data and Machine Learning on Code</p>
-                        <form action="">
-                            <!-- <input type="text" class="form-control mb-3" placeholder="Your Email Address"> -->
-                            <button type="button" onclick="location.href='https://join.slack.com/t/sourced-community/shared_invite/enQtMjc4Njk5MzEyNzM2LTFjNzY4NjEwZGEwMzRiNTM4MzRlMzQ4MmIzZjkwZmZlM2NjODUxZmJjNDI1OTcxNDAyMmZlNmFjODZlNTg0YWM';" class="btn btn-highlight w-100">Join Us Now</button>
-                        </form>
+                        <a href="https://forum.sourced.tech" class="text-dark">
```

Comment:
I'm not sure if it clear enough if there's any actionable part in the right box. I see the code, so I know that the whole box is a link... but not sure about the UX.
![image](https://user-images.githubusercontent.com/2437584/59419430-347d1180-8dcb-11e9-9ca7-379db642eb98.png)